File: modul4/Python/PRAK404-2310817210029-Putra_Whyra_Pratama_Setiawan.py
while True:
    print("Pilih program")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")
    N = int(input("Masukkan pilihan : "))
    if 1 <= N <= 4:
        a = float(input("Masukkan nilai pertama :"))
        b = float(input("Masukkan nilai kedua :"))
        if N == 1:
            print(f"Hasil penjumlahan antara {a:.2f} dengan {b:.2f} adalah {a + b:.2f}")
        elif N == 2:
            print(f"Hasil pengurangan antara {a:.2f} dengan {b:.2f} adalah {a - b:.2f}")
        elif N == 3:
            print(f"Hasil perkalian antara {a:.2f} dengan {b:.2f} adalah {a * b:.2f}")
        else:
            if b != 0:
                print(f"Hasil pembagian antara {a:.2f} dengan {b:.2f} adalah {a / b:.2f}")
            else:
                print(f"Hasil pembagian antara {a:.2f} dengan {b:.2f} adalah tidak terdefinisi")
    elif N == 5:
        print("Terimakasih, telah menggunakan kalkulator Putra Whyra Pratama S.")
        break
    else:
        print("Input anda salah, silahkan coba lagi")

File: modul4/C/PRAK404-2310817210029-Putra_Whyra_Pratama_Setiawan.c
#include <stdio.h>
int main() {
    int N, i = 0;
    float a, b;
    while(i == 0) {
        printf("\n");
        printf("Pilih program\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");
        printf("Masukkan pilihan : ");
        scanf("%d", &N);
        if(N < 5 && N > 0) {
            printf("Masukkan nilai pertama :");
            scanf("%f", &a);
            printf("Masukkan nilai kedua :");
            scanf("%f", &b);
            if(N == 1) {
                printf("Hasil pertambahan antara %.2f dengan %.2f adalah %.2f", a, b, a+b);
            } else if(N == 2) {
                printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f", a, b, a-b);
            } else if(N == 3) {
                printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f", a, b, a*b);
            } else {
                if(b != 0) {
                    printf("Hasil pembagian antara %.2f dengan %.2f adalah %.2f", a, b, a/b);
                } else {
                    printf("Hasil pembagian antara %.2f dengan %.2f adalah tidak terdefinisi", a, b);
                }
            }
        } else if(N == 5) {
            printf("Terimakasih, telah menggunakan kalkulator Putra Whyra Pratama S.");
            break;
        } else {
            printf("Input anda salah, silahkan coba lagi");
        }
    }
}